pages.zh/common/cd.md

```diff
@@ -7,13 +7,17 @@
 
 `cd {{路径}}`
```

Comment:
Wouldn't this be `路径/到/目录` (path/to/directory) instead of `路径` (path) 
Ref: <https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md> .
```suggestion
`cd {{路径/到/目录}}`
```

Reply:
That's right. Thanks:)